### DISCUSSION TITLE
Dot matrix font

### DISCUSSION THREAD

**Discussion by kouyoumin** [gopro/labs#644] [Q&A] [answered]
H11 font used for overlay and labs message becomes as ugly as H12's after the latest update.
Is there a way to switch back to the old (but stylish) dot matrix font?

**Comment by dnewman-gpsw** [comment, marked as answer]
There was a certain coolness to the crappy dot matrix font, some on the newer support needed more character definition, so the new font was added (lacking style as you suggest.)  Currently there isn't a control, but one can be added.

**Comment by dnewman-gpsw** [comment]
Prototype of future stuff. I made some improvements to font rendering.  For larger fonts there is more precision and drop shadows to improve readability. 

![GRAB-20231119-114735a](https://github.com/gopro/labs/assets/10856084/e43f1b26-0b53-4e91-92e3-b430ada59415)
![GRAB-20231119-114356a](https://github.com/gopro/labs/assets/10856084/4c4a8c48-6011-44b0-a55a-814ad1007536)

But there we also be !MFONT=1 to used select old font for all rendering:
![GRAB-20231119-114517a](https://github.com/gopro/labs/assets/10856084/dea0c6aa-2753-4e9f-81ef-1ba27b9627c1)

**Reply by kouyoumin** [reply]
Good job!
Can't wait to try the new feature!

**Comment by Deadcode85** [comment]
What will EISF=4 and GRAB=3 will do?

**Comment by dnewman-gpsw** [comment]
Nothing on current cameras. GRAB is a screen grab for capturing Labs messages at boot.  More of a developer utility, but could be helpful for support.  EISF maybe in future builds, but doesn't do anything useful yet, EIS Flags just a way to deliver settings to the stabilizer.
